### ISSUE TITLE
Docker build fails with `undefined reference to symbol 'omp_get_max_threads@@OMP_1.0'`

### ISSUE THREAD

**Issue by kmatzen** [openMVG/openMVG#2036]
Repro steps:
1. Checkout develop branch (currently f4161ef98fa676ed8b615194fae2089919a631d3)

2. `docker build . `

Output:

```
-- Found OpenMP: TRUE (found version "4.5")
```

...

```
** Enable OpenMP parallelization: ON
```

...

```
[ 61%] Linking CXX executable ../../Linux-x86_64-RELEASE/openMVG_test_rotation_averaging
/usr/bin/ld: ../../Linux-x86_64-RELEASE/libopenMVG_multiview.a(rotation_averaging_l2.cpp.o): undefined reference to symbol 'omp_get_max_threads@@OMP_1.0'
//usr/lib/x86_64-linux-gnu/libgomp.so.1: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
openMVG/multiview/CMakeFiles/openMVG_test_rotation_averaging.dir/build.make:102: recipe for target 'Linux-x86_64-RELEASE/openMVG_test_rotation_averaging' failed
make[2]: *** [Linux-x86_64-RELEASE/openMVG_test_rotation_averaging] Error 1
make[1]: *** [openMVG/multiview/CMakeFiles/openMVG_test_rotation_averaging.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
CMakeFiles/Makefile2:3487: recipe for target 'openMVG/multiview/CMakeFiles/openMVG_test_rotation_averaging.dir/all' failed
[ 61%] Linking CXX executable ../../Linux-x86_64-RELEASE/openMVG_test_solver_translation_knownRotation_kernel
[ 61%] Built target openMVG_test_solver_translation_knownRotation_kernel
[ 62%] Linking CXX executable ../../Linux-x86_64-RELEASE/openMVG_test_solver_resection_kernel
[ 62%] Built target openMVG_test_solver_resection_kernel
[ 63%] Linking CXX executable ../../Linux-x86_64-RELEASE/openMVG_test_translation_averaging
/usr/bin/ld: ../../Linux-x86_64-RELEASE/libopenMVG_multiview.a(translation_averaging_solver_l2_chordal.cpp.o): undefined reference to symbol 'omp_get_max_threads@@OMP_1.0'
//usr/lib/x86_64-linux-gnu/libgomp.so.1: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
openMVG/multiview/CMakeFiles/openMVG_test_translation_averaging.dir/build.make:102: recipe for target 'Linux-x86_64-RELEASE/openMVG_test_translation_averaging' failed
make[2]: *** [Linux-x86_64-RELEASE/openMVG_test_translation_averaging] Error 1
make[1]: *** [openMVG/multiview/CMakeFiles/openMVG_test_translation_averaging.dir/all] Error 2
CMakeFiles/Makefile2:3646: recipe for target 'openMVG/multiview/CMakeFiles/openMVG_test_translation_averaging.dir/all' failed
Makefile:140: recipe for target 'all' failed
make: *** [all] Error 2
The command '/bin/sh -c mkdir /opt/openMVG_Build;   cd /opt/openMVG_Build;   cmake -DCMAKE_BUILD_TYPE=RELEASE     -DCMAKE_INSTALL_PREFIX="/opt/openMVG_Build/install"     -DOpenMVG_BU
ILD_TESTS=ON     -DOpenMVG_BUILD_EXAMPLES=OFF     -DFLANN_INCLUDE_DIR_HINTS=/usr/include/flann     -DLEMON_INCLUDE_DIR_HINTS=/usr/include/lemon     -DCOINUTILS_INCLUDE_DIR_HINTS=/usr
/include     -DCLP_INCLUDE_DIR_HINTS=/usr/include     -DOSI_INCLUDE_DIR_HINTS=/usr/include     ../openMVG/src;     make -j 4;' returned a non-zero code: 2
```

**Comment by pmoulon** [comment]
Hi @kmatzen 
Thank you for the bug report 
Is this is happening only for unit test or for all binaries?
You can try to disable unit test build for a quick test ..

**Comment by pmoulon** [comment]
I was also able to repro the problem and fix it with a97bf6ab09e5783d86e0eda8cf160060b30b99a4
Thx you @kmatzen , @AsgeirNoM for reporting the issue, testing and helping having a fix in ;-)

**Comment by pmoulon** [comment]
@kmatzen Were you able to use the docker build now?
